CHANGELOG.md

```diff
@@ -12,6 +12,11 @@ and changes prior to that are (unfortunately) done retrospectively. Critical ite
  - migration guidance (how to convert images?)
  - changed behaviour (recipe sections work differently)
 
+## [current master](https://github.com/bleuchien/singularity)
```

Comment:
Hmmm, this doesn't seem right @bleuchien. I think it should maybe be `[current master](https://github.com/singularityware/singularity)` instead ;)

Reply:
Sorry! Copy and paste error. It's fixed.